### PR TITLE
[schemes] add two properties for elevation overlay color in color

### PR DESCRIPTION
**Pull request by fluxminds**
This PR is opened for supporting elevation overlay color for Dark Mode.

### CODE REVIEW

File: components/schemes/Color/src/MDCSemanticColorScheme.h [hunk 1]
```diff
@@ -74,6 +74,18 @@
  @c backgroundColor.
  */
 @property(nonnull, readonly, copy, nonatomic) UIColor *onBackgroundColor;
+
+/**
+ A color used on the overlay when a surface is elevated.
```
Comment (yarneo) [inline components/schemes/Color/src/MDCSemanticColorScheme.h]: A color that is used for the elevation overlays, applied with an alpha on top of surfaces.

File: components/schemes/Color/src/MDCSemanticColorScheme.h [hunk 2]
```diff
@@ -74,6 +74,18 @@
...
+ */
+@property(nonnull, readonly, copy, nonatomic) UIColor *elevationOverlayColor;
+
+/**
+ A flag that indicates whether the overlay color should be applied on an elevated surface in Dark
```
Comment (yarneo) [inline components/schemes/Color/src/MDCSemanticColorScheme.h:84]: Note: There is no elevation overlay applied outside of Dark Mode.

**Review by yarneo — APPROVED** [review]
Just a few copy edits. overall LGTM

File: components/schemes/Color/src/MDCSemanticColorScheme.h (hunk 1 above)
Comment (fluxminds) [reply components/schemes/Color/src/MDCSemanticColorScheme.h]: Addressed in 0ee543f80c066811a44a4e8b54b788e22f2c0c4a

File: components/schemes/Color/src/MDCSemanticColorScheme.h (hunk 2 above)
Comment (fluxminds) [reply components/schemes/Color/src/MDCSemanticColorScheme.h:84]: Addressed in 0ee543f80c066811a44a4e8b54b788e22f2c0c4a

**Pull request merged**